### PR TITLE
feat(components): i18n

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
No i18n

**What is the chosen solution to this problem?**
* i18n via i18next
* storybook serve / (containing locales/)
* storybook copy locales/ on build

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[x] This PR introduces a breaking change**
To use the next version, you will have to configure i18next on your app

### CODE REVIEW

File: packages/components/stories/config/i18n.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+
+i18n
+	.use(XHR)
```
Comment (jmfrancois) [inline packages/components/stories/config/i18n.js]: it should work by default without the xhr, we must provide default traduction without traduction. So even bad english will be fixed by the translations :)

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 2]
```diff
@@ -179,27 +182,28 @@ User.propTypes = {
...
 }
-
-Products.propTypes = {
+PureProducts.propTypes = {
 	renderers: React.PropTypes.shape({
 		ActionDropdown: React.PropTypes.func,
 	}),
+	t: React.PropTypes.func.isRequired,
 };
+const Products = translate('tui-components-headerBar')(PureProducts);
```
Comment (jmfrancois) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: we should be able to only translate the higher components and spread t props in it

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 3]
```diff
@@ -41,21 +41,22 @@ function Logo({ isFull, renderers, ...props }) {
 				bsStyle="link"
 				className={actionClassName}
 				hideLabel
-				label="Go to Portal"
+				label={t('GO_PORTAL')}
```
Comment (jmfrancois) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: t('GO_PORTAL', 'Go to Portal') with a defaultTranslate function which support this second argument

**Comment by jsomsanith** [comment]
Close this PR as it was a POC

**Pull request closed without merging**